templates/overrides/semantic/webbasic/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,103 @@
+﻿@using System.Security.Claims
+<!doctype html>
+<html lang="">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <title>@ViewData["Title"] - SemanticUI</title>
+
+    <environment names="Development">
+        <link rel="stylesheet" href="~/lib/semantic/dist/semantic.css" />
+        <link rel="stylesheet" href="~/css/site.css" />
+    </environment>
+    <environment names="Staging,Production">
+        <link rel="stylesheet" href="//oss.maxcdn.com/semantic-ui/2.1.8/semantic.min.css"
```

Comment:
It has to be full link with https in templates:
https://github.com/OmniSharp/generator-aspnet/commit/73b82809c1694ad69b66532cb04eac7b1540c865
I know it will hurt developers when working locally, but that's requirement:
https://twitter.com/paul_irish/status/588502455530311680?lang=en